service/src/main/java/tds/config/repositories/impl/AssessmentWindowQueryRepositoryImpl.java

```diff
@@ -65,8 +65,8 @@ public AssessmentWindowQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbc
             "        THEN UTC_TIMESTAMP() \n" +
             "        ELSE ( W.endDate + INTERVAL :shiftWindowEnd DAY) \n" +
             "    END    \n" +
-            "AND (M.sessionType = -1 OR M.sessionType = :sessionType) \n" +
-            "AND (W.sessionType = -1 OR W.sessionType = :sessionType);";
+            "AND (M.sessionType = -1 OR M.sessionType = 0) \n" +
```

Comment:
Should this be a common static property so it's very clear what it is?  `SessionType.ONLINE`